db/schema.rb

```diff
@@ -86,6 +86,7 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "description"
+    t.boolean  "billed"
```

Comment:
We should probably add an index on this (we can measure this first but just so we don't forget)